.github/workflows/km-ci-workflow.yaml

```diff
@@ -514,6 +514,41 @@ jobs:
         echo "${rname}" | grep -q -e '.*\.kontain\.KKM$' || exit 1
         echo "Kontain Runtime OK".
 
+  codeql-scan:
+    # CodeQL Code Scanning/Static Analysis
+    name: Analyze
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+
+    container:
+      image: kontainapp/buildenv-km-fedora:latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'cpp' ]
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    # Initializes the CodeQL tools for scanning.
+    - name: Initialize CodeQL
+      uses: github/codeql-action/init@v1
+      with:
+        languages: ${{ matrix.language }}
+        # TODO: Enable enhanced security checks.
+        # queries: security-extended
+
+    - run: |
+       make -C km all
```

Comment:
Maybe -j to speed things up?

Reply:
It's in parallel with everything else so it tends to be the first thing that completes, even without -j.